pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -128,11 +128,6 @@ social:
   %br/
 
   .row
-    .col-xs-12
-      %div{style: 'text-align: center; margin: 3em 0;'}
-        %h2=hoc_s(:highlights_hoc_2014_heading)
```

Comment:
This string doesn't exist and no header is being used above the images, so I just removed it.